jdaviz/configs/cubeviz/plugins/moment_maps/moment_maps.py

```diff
@@ -24,6 +25,22 @@
 @tray_registry('cubeviz-moment-maps', label="Moment Maps")
 class MomentMap(PluginTemplateMixin, DatasetSelectMixin, SpectralSubsetSelectMixin,
                 AddResultsMixin):
+    """
+    See the :ref:`Moment Maps Plugin Documentation <moment-maps>` for more details.
+
+    Only the following attributes and methods are available through the
```

Comment:
Maybe this section should be a sub-section on https://jdaviz--1642.org.readthedocs.build/en/1642/cubeviz/plugins.html#moment-maps instead of buried in API docstring, if you truly want this to be easily discoverable and used by a typical user.

I also wonder if there should be more explanation of *how* these APIs are meant to be used. Maybe in a concept notebook?

Reply:
I think we should have a broader discussion on this - I don't want to "muddy" up the plugin docs too much with API discussion that is really for more advanced users.  But we could have links from the plugin sections to their respective API docs and to https://jdaviz.readthedocs.io/en/latest/plugin_api.html.  And/or we could have an index of API calls linked from https://jdaviz.readthedocs.io/en/latest/plugin_api.html.  But I think over time, example notebooks will start using these - I don't think we need more dedicated examples just for the purpose of covering them (since all functionality exists in the UI).